src/Stackctl/FilterOption.hs

```diff
@@ -6,6 +6,7 @@ module Stackctl.FilterOption
   , filterOption
   , filterOptionFromPaths
   , filterOptionFromText
+  , filterOptionToFilePaths
```

Comment:
Should probably be `ToPaths`, not `ToFilePaths`, since I used `FromPaths` above.